velox/common/encode/Base64.cpp

```diff
@@ -305,6 +305,16 @@ void Base64::decode(const char* data, size_t size, char* output) {
   Base64::decode(data, size, output, out_len);
 }
 
+void Base64::decode_url(const char* data, size_t size, char* output) {
```

Comment:
There are few decode_url functions below around line 445. Can these functions be consolidated with those ?

Reply:
consolidated